sys/shell/commands/sc_gnrc_icmpv6_echo.c

```diff
@@ -242,8 +242,9 @@ static int _configure(int argc, char **argv, _ping_data_t *data)
     if (res != 0) {
         _usage(cmdname);
     }
+    data->id ^= (xtimer_now_usec() & UINT16_MAX);
```

Comment:
`data->id` is initialized in l. 94 to `DEFAULT_ID`, `xtimer_now_usec()` is also used in l. 311, so this does not change any dependencies, nor should it have much impact on the code size.